tests/kafkatest/services/verifiable_client.py

```diff
@@ -142,10 +142,10 @@
 def create_verifiable_client_implementation(context, parent):
     """Factory for generating a verifiable client implementation class instance
 
-    :param parent: parent class instance, either VerifiableConsumer or VerifiableProducer
+    :param parent: parent class instance, either VerifiableConsumer, VerifiableProducer or VerifiableShareGroup
```

Comment:
These references to `VerifiableShareGroup` and `ShareGroup` should probably be `VerifiableShareConsumer` and `ShareConsumer`.